### ISSUE TITLE
Update peerDependencies for eslint@1.0.0

### ISSUE THREAD

**Issue by mxdubois** [yannickcr/eslint-plugin-react#178]
eslint released 1.0.0 so it's time to bump the peerDependency.

I got this error when performing a fresh install of `eslint` and `eslint-plugin-react`:
```
/Users/XXXX/dev/ls-portal
└─┬ eslint-plugin-react@3.1.0
  └── UNMET PEER DEPENDENCY eslint@>=0.8.0 || ~1.0.0-rc-0

npm WARN ENOENT ENOENT, open '/Users/XXXXX/dev/ls-portal/package.json'
npm WARN EPEERINVALID eslint-plugin-react@3.1.0 requires a peer of eslint@>=0.8.0 || ~1.0.0-rc-0 but none was installed.
```

(using `npm@3.2.0` in case it matters, but I don't think it would)

**Comment by yannickcr** [comment]
The specified range should also match `eslint@1.0.0`. But since `npm@3.0.0` [peerDepencencies are no longer installed automatically](https://github.com/npm/npm/releases/tag/v3.0.0), you have to do it manually. I think that's why you get this warning.

**Comment by dozoisch** [comment]
@yannickcr With eslint 1.1.0, it doesn't match anymore though!

**Comment by tleunen** [comment]
You should remove the peerDep now :)

**Comment by yannickcr** [comment]
@dozoisch Actually, [according to semver](http://semver.npmjs.com) the `>=0.8.0 || ~1.0.0-rc-0` range should also match `eslint@1.1.0`. But with `npm@3.2.2` I still get an `EPEERINVALID` warning when installing eslint-plugin-react in global with `eslint@1.1.0`. No error with a local installation though. (npm bug ?)

@tleunen Yep, I'm agree. It does more harm than good right now.

**Comment by dozoisch** [comment]
So weird... Seems like a bug!

**Comment by madcapnmckay** [comment]
We're also blocked from updating to v1.1.0. Any chance we can get this dependency updated?

NPM v2.13.3
